src/confluent_kafka/admin/_config.py

```diff
@@ -18,6 +18,32 @@
 from ._resource import ResourceType
 
 
+class AlterConfigOpType(Enum):
```

Comment:
We can just call `configuration` instead of `configuration entry`

Reply:
It's better to specify to distinguish from the config resource

Reply:
We are not using ConfigEntry anywhere in Incremental alter configs implementation. Wouldn't it confuse users from docs perspective? They are not aware of ConfigEntry in the first place.

Reply:
I could use `configuration property`, it seems more used here:
```python
        self.name = name
        """Configuration property name."""
        self.value = value
        """Configuration value (or None if not set or is_sensitive==True)."""
        self.source = source
        """Configuration source."""
        self.is_read_only = bool(is_read_only)
        """Indicates whether the configuration property is read-only."""
        self.is_default = bool(is_default)
        """Indicates whether the configuration property is using its default value."""
```